include/blast/math/algorithm/Tile.hpp

```diff
@@ -8,6 +8,10 @@
 #   include <blast/math/algorithm/arch/avx2/Tile.hpp>
 #endif
 
+#if XSIMD_WITH_NEON64
+#   include <blast/math/algorithm/arch/neon64/Tile.hpp>
+#endif
+
```

Comment:
Maybe use #if, #elif, #else and #error for more robust error messages.

Reply:
Yes. But on the other hand, this `#if`-`#elif` thingy is already in `xsimd`. And if it happens that different architecture constants are defined at the same time, then you just have different specializations of `tile()` declared. The architecture is a parameter of `tile()`, so they will not conflict. And if you require to instantiate `blast::tile()` with `Arch` template parameter for which `detail::tile<...>()` does not exist, the compiler should give a pretty clear message IMO. 

What do you think?

Reply:
ok :+1: